pkg/rtc/room.go

```diff
@@ -1525,15 +1517,7 @@ func (r *Room) addParticipantWorkerLocked(p types.LocalParticipant) *participant
 func (r *Room) removeParticipantWorkerLocked(p types.LocalParticipant) {
 	identity := p.Identity()
 	if pw, ok := r.participantWorkers[identity]; ok {
-		n := len(pw.participants)
-		for idx, participant := range pw.participants {
-			if p == participant {
-				pw.participants[idx] = pw.participants[n-1]
-				pw.participants = pw.participants[:n-1]
```

Comment:
to avoid a leak here we would want to set `pw.participants[n-1]` to nil but i think we can just use a ref count here?

Reply:
yeesh, missed that too, good catch once more!